### PR TITLE
Create LabelOptout component

### PR DESCRIPTION
**Pull request by guergana**

### CODE REVIEW

File: public/i18n/qqq.json [hunk 1]
```diff
@@ -26,5 +26,6 @@
 	"query-builder-value-type-relation-dropdown-without": "Text on dropdown option to have the Query Builder select items that have the given property, but not with the given value. See also {{msg-mw|query-builder-value-type-relation-dropdown-matching}} and {{msg-mw|query-builder-value-type-relation-dropdown-regardless-of-value}}",
 	"query-builder-value-type-relation-dropdown-regardless-of-value": "Text on dropdown option to have the Query Builder select items that have the given property, regardless of the value it contains. See also {{msg-mw|query-builder-value-type-relation-dropdown-matching}} and {{msg-mw|query-builder-value-type-relation-dropdown-without}}",
 	"query-builder-delete-condition": "Text for screen readers for button that deletes the condition",
-	"query-builder-limit-number-results-description": "Text between checkbox and textfield for limiting the number of results"
+	"query-builder-limit-number-results-description": "Text between checkbox and textfield for limiting the number of results",
+	"query-builder-label-opt-out": "Checkbox to select showing IDs instead of labels in search results. Selection this option can help prevent timeouts in big queries"
```
Comment (micgro42) [inline public/i18n/qqq.json]: "Label for Checkbox [...]"

"Selecting this option [...]"

**Review by Ladsgroup — DISMISSED** [review]
Name of "Output" is typod to "Optout" everywhere :D

File: tests/unit/components/LabelOptout.spec.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+Vue.use( i18n, {
+	locale: 'en',
+	messages,
+	wikilinks: true,
+} );
+
+function newStore( getters = {} ): Store<any> {
+	return new Vuex.Store( {
+		getters: {
+			property: jest.fn().mockReturnValue( jest.fn() ),
```
Comment (micgro42) [inline tests/unit/components/LabelOptout.spec.ts]: Do we really need all those getters?

**Review by micgro42 — COMMENTED** [review]
Just some minor improvements on the description for the message

**Comment by micgro42** [comment]
> Name of "Output" is typod to "Optout" everywhere :D

No, it is the component to opt out of showing the labels. Hence, LabelOptout 🙃

**Comment by Ladsgroup** [comment]
> > Name of "Output" is typod to "Optout" everywhere :D
> 
> No, it is the component to opt out of showing the labels. Hence, LabelOptout upside_down_face

ooh, okay.

**Comment by guergana** [comment]
> > Name of "Output" is typod to "Optout" everywhere :D
> 
> No, it is the component to opt out of showing the labels. Hence, LabelOptout upside_down_face

Thanks, @micgro42 Lol

**Review by micgro42 — CHANGES_REQUESTED** [review]
Thus almost ready to be merged, just the small things I mentioned above still need to be adjusted

**Review by micgro42 — APPROVED** [review]
Looks good, thank you!

**Pull request merged**